asset-transfer-secured-agreement/application-gateway-typescript/src/utils.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright IBM Corp. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import crpto from 'crypto';
+
+export const RED = '\x1b[31m\n';
+export const GREEN = '\x1b[32m\n';
+export const RESET = '\x1b[0m';
+
+export interface AssetJSON {
+    objectType: string;
+    assetID: string;
+    ownerOrg: string;
+    publicDescription: string;
+}
+
+export interface AssetPropertiesJSON {
+    objectType: string;
+    assetID: string;
+    color: string;
+    size: number;
+    salt: string;
+}
+
+export interface AssetPriceJSON {
+    assetID: string;
+    price: number;
+    tradeID: string;
+}
+
+export interface AssetPrivateData {
+    ObjectType: string;
+    Color: string;
+    Size: number;
+}
+export interface Asset {
+    AssetId: string;
+    OwnerOrg: string;
+    PublicDescription: string;
+}
+
+export interface AssetProperties {
+    AssetId: string;
+    Color: string;
+    Size: number;
+}
+
+export interface AssetPrice {
+    AssetId: string;
+    Price: number;
+    TradeId: string;
+}
```

Comment:
I really like the separation of JSON interfaces to define the shape of JSON data sent to/from transactions, and then these interfaces for the application-level view of data sent to/from the ContractWrapper class. Two things that jump out at me though:

1. The application-level interfaces use capitalised property names. The normal convention for Java/TypeScript applications is lowercase property names, so I expected to see that here.
2. These interfaces are closely related to the ContractWrapper class so they might fit better in the contractWrapper module.